smartmin/management/__init__.py

```diff
@@ -194,4 +168,3 @@ def check_all_permissions(sender, **kwargs):
 
 post_migrate.connect(check_all_permissions)
 post_migrate.connect(check_all_group_permissions)
```

Comment:
Is there a better hook for all this stuff now in Django 1.10? It is one reason our migrates/syncdbs are slow is that we evaluate this way too much just to be sure it is actually called